### ISSUE TITLE
add toolkit/tests/create_molecules.py

### ISSUE THREAD

**Issue by adalke** [openforcefield/openff-toolkit#959]
The test suite takes a while to run manually, even if only one test module/function is to be run. This is because much of the data sets and modules are processed at import time. For example, test_toolkits.py and test_topology.py both import test_forcefield.py for the following:
```
from openff.toolkit.tests.test_forcefield import (
    create_acetaldehyde,
    create_acetate,
    create_cyclohexane,
    create_ethanol,
    create_reversed_ethanol,
)
```

test_forcefield imports many modules and data that test_toolkits and test_forcefield don't need.

I plan to move those "create_*" functions into a new module, tests/create_molecules.py so those two test modules don't need to bring in the entire system that test_forcefield needs.

My tests show that this reduces the startup time for test_toolkits and test_forcefield by a second.

**Comment by j-wags** [comment]
This is a great idea. 

For tests that shouldn't require OpenEye or RDKit, we've building molecules directly using the `Molecule.add_atom` and `Molecule.add_bond`. We did this first in `test_forcefield`, but these molecules have since started being used in other test files, so I think that moving these into their own file will be a good change.

**Comment by j-wags** [comment]
This has been resolved by #963
